forums/views.py

```
@@ -0,0 +1,100 @@
+        course_slug = self.kwargs.get('course_slug', '')
+        course = get_object_or_404(Course, slug=course_slug)
+        
+        # Check if this is a topic submission or a reply submission
+        if 'topic_id' in request.POST:
+            # This is a reply submission
+            return self.handle_reply_submission(request, course_slug)
+        else:
+            # This is a new topic submission
+            return self.handle_topic_submission(request, course, course_slug)
+    
+    def handle_topic_submission(self, request, course, course_slug):
```

Ruff (0.8.2)

Missing return type annotation for public function `handle_topic_submission`

(ANN201)